modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-6_{context}"]
+= Release notes for {pipelines-title} General Availability 1.6
+
+With this update, {pipelines-title} General Availability (GA) 1.6 is available on {product-title} 4.7, 4.8 and 4.9.
```

Comment:
1.6 is only going to be available on 4.9 though
cc @siamaksade

Reply:
Updated. Thanks.